lib/queuePopulator/KafkaLogConsumer/LogConsumer.js

```diff
@@ -0,0 +1,169 @@
+const kafka = require('node-rdkafka');
+const async = require('async');
+const ListRecordStream = require('./ListRecordStream');
+
+// maximum time to wait for consumer group to rebalance (ms)
+const maxDelayRebalance = 120000;
```

Comment:
It can improve readability to attach the unit to the variable name:

```suggestion
const maxDelayRebalanceMs = 120000;
```